plugins/ipqualityscore/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM rapid7/insightconnect-python-3-38-plugin:4
```

Comment:
Is it possible to update this to version 5 the newest version of insightconnect plugin?
`FROM rapid7/insightconnect-python-3-38-plugin:5`